docs/remote.rst

```diff
@@ -10,7 +10,7 @@ tests that you can successfully run locally. And you
 have a ssh-reachable machine ``myhost``.  Then
 you can ad-hoc distribute your tests by typing::
 
-    pytest -d --tx ssh=myhostpopen --rsyncdir mypkg mypkg
```

Comment:
the --rsyncdir accept the directory, the last "mypkg" is not needed and misleading.